tools/autograd/derivatives.yaml

```diff
@@ -884,6 +884,9 @@
 - name: embedding_renorm_(Tensor self, Tensor indices, double max_norm, double norm_type)
   self: not_implemented("embedding_renorm")
 
+- name: no_grad_embedding_renorm_(Tensor self, Tensor indices, double max_norm, double norm_type)
+  output_differentiability: [False, False, False, False]
```

Comment:
This is wrong: output_differentiability can only be used if the inputs _definitely_ won't come into contact with autograd. Running the following script on master with these changes,
```
import torch
import torch.nn.functional as F

input = torch.tensor([[1,2,4,5],[4,3,2,9]])
embedding_matrix = torch.randn(10, 3)

var1 = torch.randn(10, 3, requires_grad=True)
var2 = var1.detach().requires_grad_()
output1 = var1 * embedding_matrix
output2 = var2 * embedding_matrix

output1.sum().backward()

ignore = F.embedding(input, embedding_matrix, max_norm=0.01)

output2.sum().backward()

print(var1.grad)
print(var2.grad)
```
var1.grad and var2.grad are different. The expected result should be because F.embedding modified the weight matrix, output2.backward() should complain about its dependencies having been modified and should not fail silently like this.

Reply:
Fix here: 

https://github.com/pytorch/pytorch/pull/14639